libfuse/fs_darwin.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libfuse
+
+import (
+	"bazil.org/fuse"
+	"bazil.org/fuse/fs"
+	"golang.org/x/net/context"
+)
+
+func (r *Root) platformLookup(ctx context.Context, req *fuse.LookupRequest, resp *fuse.LookupResponse) fs.Node {
+	switch req.Name {
+	case VolIconFileName:
+		volIcon, err := newExternalBundleResourceFile("KeybaseFolder.icns")
+		if err != nil {
+			r.log().CErrorf(ctx, "Error getting bundle resource path: %s", err)
+		} else if volIcon != nil {
```

Comment:
I think you can just do `return volIcon, nil` here. Since these files never change, we can leave `EntryValid` to the default, I think. (setting it to 0 is only for special files that can be updated at any time)